src/sentry/tasks/weekly_escalating_forecast.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+from typing import Any, Dict, List, Tuple
+
+from django.db import transaction
+
+from sentry.issues.escalating_issues_alg import issue_spike
+from sentry.models import Group, GroupStatus
+from sentry.models.groupforecast import GroupForecast
+from sentry.models.groupsnooze import GroupSnooze
+from sentry.tasks.base import instrumented_task
+from sentry.utils.json import JSONData
+
+BATCH_SIZE = 200
+GroupForecastTuple = Tuple[Group, List[int]]
+
+
+def query_groups_past_counts(groups: List[Group]) -> JSONData:
+    return {}
+
+
+def get_forecast_per_group(
+    group_forecast_list: List[GroupForecastTuple],
+    archived_groups: List[Group],
+    response: JSONData,
+) -> None:
+    """
+    Modifies `group_forecast_list` to contain a list of forecasted values for each group.
+
+    `group_forecast_list`: List of GroupForecastTuple to be modified
+    `archived_groups`: List of archived groups to be forecasted
+    `archived_groups`: Snuba response for group event counts
+    """
+    data_index = 0
+    time = datetime.now()
+    for group in archived_groups:
+        group_data: Dict[str, List[Any]] = {"intervals": [], "data": []}
+        while (
+            data_index < len(response["data"])
```

Comment:
FYI I'm going to be returning just "data" rather than the whole response.

There's going to be a maximum of 336 entries (14d*24h) per group.

Something like this:
```
           [
                {
                    "count()": 1,
                    "group_id": group_1_id,
                    "hourBucket": to_start_of_hour(event1.datetime),
                    "project_id": self.project.id,
                },
                {
                    "count()": 1,
                    "group_id": group_2_id,
                    "hourBucket": to_start_of_hour(event2.datetime),
                    "project_id": self.project.id,
                },
                {
                    "count()": 2,
                    "group_id": group_2_id,
                    "hourBucket": to_start_of_hour(event3.datetime),
                    "project_id": self.project.id,
                },
            ]
```